src/oc/interaction/representations/interaction.clj

```diff
@@ -1,16 +1,23 @@
 (ns oc.interaction.representations.interaction
   "Resource representations for OpenCompany interactions."
-  (:require [defun.core :refer (defun defun-)]
+  (:require [clojure.string :as string]
+            [defun.core :refer (defun defun-)]
             [cheshire.core :as json]
             [oc.lib.hateoas :as hateoas]
-            [oc.interaction.config :as config]))
+            [oc.interaction.config :as config]
+            [oc.interaction.resources.interaction :as interact-res]))
 
 (def comment-media-type "application/vnd.open-company.comment.v1+json")
 (def comment-collection-media-type "application/vnd.collection+vnd.open-company.comment+json;version=1")
 
 (def reaction-media-type "application/vnd.open-company.reaction.v1+json")
 
-(def representation-props [:body :reaction :author :created-at :updated-at])
+(def representation-props [:uuid :body :reaction :author :reactions :created-at :updated-at])
+
+(defn- map-kv
```

Comment:
Is this or something like it in the oc.lib ??  @belucid

Reply:
The reason I ask is this function was borrowed from the storage service.

Reply:
There isn't, and there isn't right now a ns in oc.lib for truly general purpose stuff like this. I'm not opposed to factoring this function out of storage and here into oc.lib, but it's also just a 1-liner so I don't know that it's a huge win either.